pkg/task/runner.go

```diff
@@ -174,14 +174,14 @@ func (m *TaskGroupRunner) rollback() {
 		err := m.rollbacks[i].ExecuteIt()
 		if err != nil {
 			// warn this rollback error & continue with the next rollbacks
-			glog.Warningf("failed to rollback run task: '%s': error '%s'", m.rollbacks[i], err.Error())
+			glog.Warningf("failed to rollback run task {%s}: %s", m.rollbacks[i].Runtask.Name, err.Error())
```

Comment:
Is `err.Cause.Error()` better than `err.Error()`?
Is there any chance of nil pointer?
cc @imazik

Reply:
We can use `errors.Cause(err)`. By the description of `Cause()` it will return the cause if present or the original error itself.